### ISSUE TITLE
how to get the style_mixing_example like the official version?

### ISSUE THREAD

**Issue by mumuyanyan** [rosinality/stylegan2-pytorch#40]
how to get the style_mixing_example like the official version?

**Comment by rosinality** [comment]
You can pass multiple noises to generator and set inject_index like `g_ema([noise1, noise2], inject_index=3)`

**Comment by DevD1092** [comment]
What should `noise1` and `noise2` correspond to? Can they be any random noises? What should be the dimension, etc.? Could you please provide an example? Thank you

**Comment by rosinality** [comment]
noise1 is noise input used before inject_index, and noise2 is used after it. You can generate noise for generator using torch.randn(batch_size, 512).

**Comment by DevD1092** [comment]
@rosinality : Adding to the above question. Let' say one of the source (let's assume `noise1`) is to be taken from an **external image** and not produced through normal random distribution. The other source `noise2` can be produced through `torch.randn`. Then how should the `noise1` be generated (PCA)? Basically, one of the styles for the mixing is to be taken externally.

**Comment by rosinality** [comment]
@DevD1092 Do you mean how you can project images to noise or latent spaces?

**Comment by DevD1092** [comment]
> @DevD1092 Do you mean how you can project images to noise or latent spaces?

Yes. If I have an external image (not generated by the generator). Can I get the correspondence of the noise for that image?

**Comment by rosinality** [comment]
@DevD1092 You can use projector.py to project images to latent spaces.
